DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿using DepositeCalcTests.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class CalculatorPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Calculator";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "", "")]
+        [TestCase("", "", "1")]
+        [TestCase("", "1", "")]
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
+        public void MandaroryTextFieldsWith365BtnTest(string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.MandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.ClickOnFinancialYear365RadioBtn();
+
+            // Assert
+            Assert.IsTrue(calculatorPage.GetCalculateBtnCurrentStatus(), "Calculate button is clickable. This means that one of the text fields or one of the Financial year radio buttons are not mandatory");
+        }
+
+        [TestCase("", "", "")]
+        [TestCase("", "", "1")]
+        [TestCase("", "1", "")]
+        [TestCase("", "1", "1")]
+        [TestCase("1", "", "")]
+        [TestCase("1", "", "1")]
+        [TestCase("1", "1", "")]
+        public void MandaroryTextFieldsWith360BtnTest(string depositAmount, string interestRate, string investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.MandatoryTextFields(depositAmount, interestRate, investmentTerm);
+            calculatorPage.ClickOnFinancialYear360RadioBtn();
+
+            // Assert
+            Assert.IsTrue(calculatorPage.GetCalculateBtnCurrentStatus(), "Calculate button is clickable. This means that one of the text fields or one of the Financial year radio buttons are not mandatory");
+        }
+
+        [TestCase(1, 1, 1)]
+        [TestCase(10, 10, 1)]
+        [TestCase(99000, 10, 1)]
+        [TestCase(100000, 10, 1)]
+        [TestCase(10, 99, 1)]
+        [TestCase(10, 100, 1)]
+        [TestCase(10, 50, 364)]
+        [TestCase(10, 50, 365)]
+        public void ValidCalculation365Test(double depositAmount, double interestRate, double investmentTerm)
```

Comment:
Also, you can calculate the expected values and add them as parameters.
```suggestion
        [TestCase(100000, 100, 365, "100 000.00", "200 000.00"]
        public void ValidCalculation365Test(double depositAmount, double interestRate, double investmentTerm, string expectedIncome, string expectedInterest)
```